### PR TITLE
Retranslate sequential.md

### PR DESCRIPTION
**Pull request by EthanJYK**
#86 
[For Reviewers Proficient in Korean]
- framework-native tensors / TensorFlow data tensors: 이미 각 백엔드를 통해 텐서로 정의되어 있는 데이터들을 의미합니다. 이 경우 Keras는 placeholder를 생성하지 않고 곧바로 Tensor를 읽어들입니다. `프레임워크(예: TensorFlow)를 통해 이미 정의된 텐서`로 번역하였습니다.
- `fit`의 validation_steps: 원문에 두 가지 경우가 별도로 서술되어 있습니다. 이 둘을 하나의 항목으로 합치되, 안에서 두 개의 경우로 분리하였습니다.
- `fit`의 y: x의 경우 설명이 항목별로 나뉘어 구성된 반면 y의 경우 전체가 한 문단으로 작성되어 있습니다. 내용상 큰 차이가 없기 때문에 x와 같은 형식으로 통일하여 항목별로 작성합니다.
- batch_size 항목들: 흐름이 자연스럽도록 문장의 순서를 살짝 조정합니다.
- sample_weight, class_weight, sample_weight_mode 항목들: 설명을 보다 자세하게 덧붙여서 문장을 재구성했습니다.
- 그 밖에 원문에 존재하는 오류는 PR을 통해 원문 수정 요청하면서 본문에도 정정하였습니다. 자세한 사항은 https://github.com/keras-team/keras/pull/13422 참고바랍니다.
- 전체 용어 및 양식은 Keras Korea 번역팀의 합의안을 따릅니다.

### CODE REVIEW

File: sources/models/sequential.md [hunk 1]
```diff
@@ -1,645 +1,318 @@
...
 ### compile
-
-
 ```python
 compile(optimizer, loss=None, metrics=None, loss_weights=None, sample_weight_mode=None, weighted_metrics=None, target_tensors=None)
 ```
+학습시킬 모델을 구성합니다.
 
+__인자__
+- __optimizer__: 모형에 사용할 최적화 함수<sub>Optimizer</sub>를 지정합니다. 케라스가 제공하는 최적화 함수의 이름 문자열<sub>String</sub> 또는 개별 최적화 함수의 인스턴스를 입력합니다. 자세한 사항은 [최적화 함수](/optimizers)를 참고하십시오.
```
Comment (fuzzythecat) [inline sources/models/sequential.md]: 모형에 →  학습에로 바꾸면 어떨까요!

File: sources/models/sequential.md [hunk 2]
```diff
@@ -1,645 +1,318 @@
...
+- __loss__: 모형에 사용할 손실 함수<sub>Loss function</sub>를 지정합니다. 케라스가 제공하는 손실 함수의 이름 문자열 또는 개별 손실 함수의 인스턴스를 입력합니다. 자세한 사항은 [손실 함수](/losses)을 참고 하십시오. 모델이 다중의 결과<sub>Output</sub>를 출력하는 경우, 손실 함수를 리스트 또는 딕셔너리 형태로 입력하여 결과의 종류별로 서로 다른 손실 함수를 적용할 수 있습니다. 이 경우 모델이 최소화할 손실값은 모든 손실 함수별 결과값의 합이 됩니다.
```
Comment (fuzzythecat) [inline sources/models/sequential.md]: 모형에 → 학습에로 바꾸면 어떨까요!

Comment (fuzzythecat) [reply sources/models/sequential.md]: 다중의 결과<sub>Output</sub>를 출력하는 경우 → 두 개 이상의 출력값을 가질 경우는 어떨까요

File: sources/models/sequential.md [hunk 3]
```diff
@@ -1,645 +1,318 @@
...
+    - NumPy 배열<sub>array</sub> 또는 배열과 같은 형식의 데이터. 다중 입력의 경우 배열의 리스트.
+    - 모델이 이름이 지정된 입력값을 받는 경우 이름과 배열/텐서가 `key`와 `value`로 연결된 딕셔너리.
+    - 파이썬 제너레이터 또는 `keras.utils.Sequence` 인스턴스로 `(inputs, targets)` 혹은 `(inputs, targets, sample weights)`를 반환하는 것.
+    - 프레임워크(예: TensorFlow)를 통해 이미 정의된 텐서를 입력받는 경우 `None`(기본값).
+- __y__: 목표 데이터로 다음과 같은 형태가 가능합니다.
+    - NumPy 배열 또는 배열과 같은 형식의 데이터. 다중의 결과를 출력하는 경우 배열의 리스트.
+    - 프레임워크(예: TensorFlow)를 통해 이미 정의된 텐서를 입력받는 경우 `None`(기본값).
+    - 결과값의 이름이 지정되어 있는 경우 이름과 배열/텐서가 `key`와 `value`로 연결된 딕셔너리.
+    - 만약 `x`에서 파이썬 제너레이터 또는 `keras.utils.Sequence` 인스턴스를 사용할 경우 목표값이 `x`와 함께 입력되므로 별도의 `y`입력은 불필요합니다.
+- __batch_size__: `int` 혹은 `None`. 역전파<sub>Backpropagation</sub> 과정에서 한 번에 그래디언트가 적용될 표본의 개수입니다. 따로 정하지 않는 경우 `batch_size`는 기본값인 32가 됩니다. 별도의 심볼릭 텐서나 제네레이터, 혹은 `Sequence` 인스턴스로 데이터를 받는 경우 인스턴스가 자동으로 배치를 생성하기 때문에 별도의 `batch_size`를 지정하지 않습니다.
```
Comment (fuzzythecat) [inline sources/models/sequential.md]: 역전파<sub>Backpropagation</sub> 과정에서 한 번에 그래디언트가 적용될 표본의 개수입니다에 약간의 수정이 필요해 보입니다.

gradient update는 mini-batch 데이터로 계산한 gradient를 바탕으로 모델의 가중치를 update하는 과정이기 때문에, 해당 과정이 더 명확하게 드러나도록 수정 부탁드립니다!

File: sources/models/sequential.md [hunk 4]
```diff
@@ -1,645 +1,318 @@
...
+    - 데이터 세트 또는 데이터 세트의 이터레이터. 이 경우 `validation_steps`를 같이 명시해야 합니다.
+- __shuffle__: 불리언 또는 문자열 `'batch'`. 불리언 입력의 경우 각 에폭을 시작하기 전에 훈련 데이터를 뒤섞을지를 결정합니다. `'batch'`의 경우 HDF5 데이터의 제약을 해결하기 위한 설정으로 각 배치 크기 안에서 데이터를 뒤섞습니다. `steps_per_epoch`값이 `None`이 아닌 경우 `shuffle`인자는 무효화됩니다.
+- __class_weight__: 필요한 경우에만 사용하는 인자로, 각 클래스 인덱스를 `key`로, 가중치를 `value`로 갖는 딕셔너리를 입력합니다. 인덱스는 정수, 가중치는 부동소수점 값을 갖습니다. 주로 훈련 데이터의 클래스 분포가 불균형할 때 이를 완화하기 위해 사용하며, 손실 계산 과정에서 표본 수가 더 적은 클래스의 영향을 가중치를 통해 끌어올립니다. 학습 과정에서만 사용됩니다.
+- __sample_weight__: 특정한 훈련 표본이 손실 함수에서 더 큰 영향을 주도록 하고자 할 경우에 사용하는 인자입니다. 일반적으로 표본과 동일한 길이의 1D NumPy 배열을, 시계열 데이터의 경우 `(samples, sequence_length)`로 이루어진 2D 배열을 입력하여 손실 가중치와 표본이 1:1로 짝지어지게끔 합니다. 2D 배열을 입력하는 경우 반드시 `compile()` 단계에서 `sample_weight_mode="temporal"`로 지정해야 합니다. `x`가 제너레이터 또는 `Sequence`인스턴스인 경우는 손실 가중치를 `sample_weight`인자 대신 `x`의 세 번째 구성요소로 입력해야 적용됩니다.
+- __initial_epoch__: `int`. 특정한 에폭에서 학습을 시작하도록 시작 회차를 지정합니다. 이전의 학습을 이어서 할 때 유용합니다.
+- __steps_per_epoch__: `int` 혹은 `None`. 1회의 에폭을 이루는 배치의 개수를 정의하며 기본값은 `None`입니다. 프레임워크에서 이미 정의된 텐서(예: TensorFlow data tensor)를 훈련 데이터로 사용할 때 `None`을 지정하면 자동으로 데이터의 표본 수를 배치 크기로 나누어 올림한 값을 갖게 되며, 값을 구할 수 없는 경우 `1`이 됩니다.
```
Comment (fuzzythecat) [inline sources/models/sequential.md]: data tensor는 데이터를 저장하고있는 텐서이기 때문에 데이터 텐서로 번역하면 어떨까 합니다.

File: sources/models/sequential.md (hunk 2 above)
Comment (fuzzythecat) [reply sources/models/sequential.md]: 생각해보니 두 개 이상의 출력값이라 적는 경우 multi-class 분류와 혼동될 수도 있을것 같네요

File: sources/models/sequential.md (hunk 3 above)
Comment (EthanJYK) [reply sources/models/sequential.md]: "손실로부터 그래디언트를 구하고 가중치를 업데이트하는 과정 한 번에 사용할 표본의 개수입니다." 로 수정합니다.

File: sources/models/sequential.md (hunk 4 above)
Comment (EthanJYK) [reply sources/models/sequential.md]: 수정하였습니다.

**Comment by fuzzythecat** [comment]
감사합니다!
